ui/components/component-library/text/text.scss

```diff
@@ -59,6 +59,10 @@ $text-variants: (
       @else {
         font-weight: $weight;
       }
+
+      strong {
+        font-weight: var(--font-weight-bold);
+      }
```

Comment:
A implementation of this was recently merged so you should be able to remove this now.

Reply:
thanks, nice catch! Missed that the other implementation had the rule at top of `.mm-text` and trusted the merge too much :-)